TemplateTests/TemplateTests.swift

```diff
@@ -0,0 +1,28 @@
+@testable import Template
+import XCTest
+
+final class TemplateTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
```

Comment:
Premyslim jestli tady ty examply a prazdny metody nedat doprdele? Kdyz je v kazdym projektu stejne mazeme a nechavame tam pouze ten test s `app.launch()` v UITests. 
Nebo nechali byste to tam vsechno, aby to bylo jako kdyz si clovek zalozi prazdny projekt v xcode? 🤔